.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
 
     strategy:
       matrix:
-        otp: ['24', '25', '26', '27']
+        otp: ['25', '26', '27']
         rebar: ['3.23']
```

Comment:
Since we're up'ing stuff we could use 3.24 here too 😄

Reply:
👌🏻